### DISCUSSION TITLE
Customizing AsciiDoc View Mode

### DISCUSSION THREAD

**Discussion by amelillo** [gsantner/markor#2063] [General] [answered]
Hello;
As I could see in these days, Markor at last supported AsciiDoc rendering in View Mode.
What I couldn't be able to do is messing around with an option or a css to customize View Mode and mirror the style I already use on my laptop in a variety of applications and browser extensions, so to be able to export the very same output from different devices.
At least, I would be able to change the default rendering mode which is a dark theme (bright letters on dark background) trying at least to mimic the Markdown rendering default which, in turn, is light.
What file should I mess with? Can I just load a custom css on Markor?
Or, should I suppose a customizable View Mode in a near future?

Thank you!

am

**Comment by gsantner** [comment, marked as answer]
There is a settings option Inject-head at Markor settings. There you can globally add CSS.

For classes/default configuration check the css files in this git repository.

**Reply by amelillo** [reply]
Great, I was just wondering about references. I'll maybe try to inject the css default I use, thank you.

**Comment by aisbergde** [comment]
I would suggest to implement at least the switch between dark and light theme using Markor settings.

And I am so happy, that we have built in AsciiDoc preview now!
